spec/features/venues/view_venue_details_spec.rb

```diff
@@ -21,4 +21,20 @@
     expect(page).to have_content(venue.address)
     expect(page).to have_content(venue.capacity)
   end
+
+  scenario 'If T accessible, user should see message: T is nearby' do
+    venue = FactoryGirl.create(:venue, t_accessible: true)
+
+    visit venue_path(venue)
+
+    expect(page).to have_content(venue.display_t_accessible)
```

Comment:
We were debating about this. If we check the string directly, and then we decide to change the specific string later, we end up breaking 2 tests instead of just one (the model test and the acceptance test) which leads to a fragile test. Given that, should we make the test more fragile and test the string instead?